### ISSUE TITLE
Skipping Validatable interface

### ISSUE THREAD

**Issue by rmscoal** [go-ozzo/ozzo-validation#192]
I wanted to validate a struct. For example:

```go
type User struct {
    Name string
    Cars []Car
}

type Car struct {
    ID int
    Name string
}

func (v User) Validate() error {
    return validation.ValidateStruct(&v,
        validation.Field(&v.Name, validation.Required),
        validation.Field(&v.Cars, validation.Length(1, 0)), // car's validation is triggered here... But I don't really care... How can I skip it
    )
}

func (c Car) Validate() error {
    return validation.ValidateStruct(&c,
        validation.Field(&c.Name, validation.Required),
    )
}
```

As of the documentation, `Car` struct implements the `Validatable` interface. Hence, it will be `.Validate()` will be called. But what if I want to skip it? Just the `validation.Length()`.

Thank you so much.

**Comment by kolkov** [comment]
Currently there's no built-in way to skip the `Validatable` interface when it's implemented on a type. The library automatically calls `Validate()` on any type that implements it during field validation.

One workaround is to wrap the slice in a custom rule where you only check `Length` without triggering the nested validation. Are you still running into this? If so, I can look into adding an option to opt out of automatic `Validatable` calls for specific fields.
